src/Library/demos/List Model/main.js

```diff
@@ -0,0 +1,107 @@
+import Gio from "gi://Gio";
+import Gdk from "gi://Gdk";
+import GObject from "gi://GObject";
+import Gtk from "gi://Gtk";
+import Adw from "gi://Adw";
+
+const stack = workbench.builder.get_object("stack");
+const list_box = workbench.builder.get_object("list_box");
+const flow_box = workbench.builder.get_object("flow_box");
+const add = workbench.builder.get_object("add");
+const remove = workbench.builder.get_object("remove");
+const list_box_editable = workbench.builder.get_object("list_box_editable");
+const search_entry = workbench.builder.get_object("search_entry");
+
+//Model
+const model = new Gtk.StringList();
```

Comment:
```suggestion
const model = new Gtk.StringList({
  strings: [
    "Default Item 1",
    "Default Item 2",
    "Default Item 3",
  ],
});
```

I think this is the easiest way to avoid dealing with empty states in the demo. What do you think?

Reply:
Yes, it's obviously better than a void space